buildimages.sh

```diff
@@ -100,7 +109,8 @@ fi
 # full (re)build including C library and all applications
 make clean
 build_pc98
-build_8018x
-build_8088
+build_rom_8018x
+build_rom_8088
 build_ibm
 build_ibm_all
+build_rom_swan
```

Comment:
Thanks for fixing the build_rom_\* errors. Can you move build_rom_swan to just below build_rom_8088? I want to keep build_ibm_\* as the last builds to ensure the C library and applications are built for IBM PC instead of another  architecture. This script is sometimes used outside of GitHub CI and it is convenient to know that the system is in a known IBM PC state after running.